xla/myplugin/BUILD

```diff
@@ -0,0 +1,35 @@
+load("//xla:xla.bzl", "xla_cc_binary", "xla_cc_test")
+
```

Comment:
Why did any of this require a docker container?

Reply:
Cause the docs said so.